test/script.py

```diff
@@ -58,7 +58,7 @@ class bcolors:
     try:
         if len(sys.argv) == 4:
             binary = FirefoxBinary(sys.argv[3])
-            driver = webdriver.Firefox(fp, firefox_binary=binary, log_path=os.devnull)
+            driver = webdriver.Firefox(fp, firefox_binary=binary, service_log_path=os.devnull)
         else:
             driver = webdriver.Firefox(fp, log_path=os.devnull)
```

Comment:
Add the same to this line and it's good to go. Thanks!

Reply:
Oops, didn't see that one. Done!